content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -257,7 +258,7 @@ Notes:
 
```

Comment:
Line 243:

- "See the guide to [creation of profiles]" is a bit "off" -> "Refer to this guide on [creating profiles]" or "...[creating profiles automatically]

```diff
- 1. Follow the instructions on the UI to create a namespace. See the guide to
+ 1. Follow the instructions on the UI to create a namespace. Refer to this guide on
-  [creation of profiles](/docs/components/multi-tenancy/getting-started/#automatic-creation-of-profiles).
+ [creating profiles](/docs/components/multi-tenancy/getting-started/#automatic-creation-of-profiles).
```

Line 212

- As a non-expert KF user, I and some user may want to know why I have to follow certain steps and there are a lot of them in the KF guides. For example:

```diff
- Follow these steps to access the Kubeflow central dashboard:
+ To access the Kubeflow central dashboard, follow these steps:
```

Reply:
+1, now I remember, goal -> instructions